Assets/Scripts/SS3D/Systems/Inventory/Containers/ClothesDisplayer.cs

```diff
@@ -0,0 +1,222 @@
+using SS3D.Logging;
+using SS3D.Systems.Inventory.Items;
+using UnityEngine;
+using FishNet.Object.Synchronizing;
+using UnityEditor;
+using FishNet.Object;
+
+namespace SS3D.Systems.Inventory.Containers
+{
+    /// <summary>
+    /// Display cloth on player for all clients.
+    /// </summary>
+    public class ClothesDisplayer : NetworkActor
+    {
+        /// <summary>
+        /// A small structure containing information regarding clothes on player, to help syncing them over the network.
+        /// For each bodypart that can have clothing, it also contains information on the item to display, if it should show or not.
+        /// </summary>
+        private struct ClothDisplayData
+        {
+            public ClothDisplayData(NetworkObject bodyPart, bool display, Item clothToDisplay)
+            {
+                _bodyPart= bodyPart;
+                _display= display;
+                _clothToDisplay= clothToDisplay;
+            }
+            public NetworkObject _bodyPart;
+            public bool _display;
+            public Item _clothToDisplay;
+        }
+
+        public HumanInventory _inventory;
+
+        // The root game objects for clothes
+        public Transform ClothesRoot;
+
+        // Game objects on the human prefab to display clothes.
+        public NetworkObject Hat;
+        public NetworkObject Eyes;
+        public NetworkObject Jumpsuit;
+        public NetworkObject HandLeft;
+        public NetworkObject HandRight;
+        public NetworkObject FootLeft;
+        public NetworkObject FootRight;
+        public NetworkObject Identification;
+        public NetworkObject Backpack;
+	    public NetworkObject Face;
+        public NetworkObject EarLeft;
+        public NetworkObject EarRight;
+
+        // Syncvar to sync meshes and cloth display between clients
+        [SyncVar(OnChange = nameof(SyncCloth))]
+        private ClothDisplayData _hatData;
+
+        [SyncVar(OnChange = nameof(SyncCloth))]
+        private ClothDisplayData _eyesData;
+
+        [SyncVar(OnChange = nameof(SyncCloth))]
+        private ClothDisplayData _jumpsuitData;
+
+        [SyncVar(OnChange = nameof(SyncCloth))]
+        private ClothDisplayData _handLeftData;
+
+        [SyncVar(OnChange = nameof(SyncCloth))]
+        private ClothDisplayData _handRightData;
+
+        [SyncVar(OnChange = nameof(SyncCloth))]
+        private ClothDisplayData _footLeftData;
+
+        [SyncVar(OnChange = nameof(SyncCloth))]
+        private ClothDisplayData _footRightData;
+
+        [SyncVar(OnChange = nameof(SyncCloth))]
+        private ClothDisplayData _identificationData;
+
+        [SyncVar(OnChange = nameof(SyncCloth))]
+        private ClothDisplayData _backpackData;
+
+	    [SyncVar(OnChange = nameof(SyncCloth))]
+        private ClothDisplayData _faceData;
+
+        [SyncVar(OnChange = nameof(SyncCloth))]
+        private ClothDisplayData _earLeftData;
+
+        [SyncVar(OnChange = nameof(SyncCloth))]
```

Comment:
Hardcoding these values make little sense for me, can't we have a ClothingSlot enum and use a SyncDictionary to sync these values?

Reply:
probably yes, I can try that if you prefer.